package.json

```diff
@@ -77,5 +78,6 @@
     "linux": {
       "icon": "./imgs/installer/zeal.png"
     }
-  }
+  },
+  "packageManager": "pnpm@9.7.0"
```

Comment:
Can we remove this, as it's not an actual requirement? npm and yarn should still work.